Module/Common/Functions.XccdfXml.ps1

```diff
@@ -432,18 +444,33 @@ function Split-BenchmarkId
             $returnId = "FireFox_All"
             continue
         }
-        {$PSItem -match 'Excel' -or $PSItem -match 'Outlook' -or $PSItem -match 'PowerPoint' -or $PSItem -match 'Word'}
+        {$PSItem -match 'Excel' -or $PSItem -match 'Outlook' -or $PSItem -match 'PowerPoint' -or $PSItem -match 'Word' -or $PSItem -match 'System'}
```

Comment:
This could be cleaned up with $PSItem -match 'Outlook|PowerPoint|Word|System'

Reply:
Updated